ui/package.json

```diff
@@ -74,6 +74,7 @@
     "react-resizable-panels": "2.0.22",
     "react-yjs": "2.0.0",
     "use-resize-observer": "9.1.0",
+    "y-websocket": "^2.0.4",
```

Comment:
Please use exact versioning on the frontend. (remove ^ and rerun yarn. When adding a dep you can do `yarn add -E your-dependency` to add the latest as an exact version 👍🏻